templates/csm_setup_conf_template.json

```diff
@@ -1,9 +1,9 @@
-{
-  "cluster": {
-    "cluster_id": "bae4b468-565d-49df-9495-a43a5d89babc",
-    "DEPLOYMENT": {
+{ 
+  "DEPLOYMENT": {
       "mode": "dev"
```

Comment:
For Non-dev mode what will be the value?

Reply:
@pawankumarsrivastava  For non dev mode ```"DEPLOYMENT": {
      "mode": "dev"
}``` will not be present. 
dev mode is set according to this tag